tests/jobs/design2params/test_design2params.py

```diff
@@ -432,7 +433,11 @@ def test_headers_trailing_whitespace(paramnames, tmpdir):
             0,
             "design_matrix.xlsx",
         )
-    with pytest.raises(SystemExit, match="whitespace"):
+    if version.parse(pd.__version__) < version.parse("0.25"):
```

Comment:
I wonder how long we should support pandas < 1. At least we should make an issue. I wanted to suggest a due date on said issue, but searching for the possibility of adding due dates to issues was quite depressing: https://github.com/isaacs/github/issues/432

Looks like a lot of people still want it though!

Reply:
Agreed, I think the hope is for another month. https://github.com/equinor/semeio/issues/254